resources/themes/cura-light/theme.json

```diff
@@ -321,7 +321,12 @@
         "favorites_header_hover": [245, 245, 245, 255],
         "favorites_header_text": [31, 36, 39, 255],
         "favorites_header_text_hover": [31, 36, 39, 255],
-        "favorites_row_selected": [196, 239, 255, 255]
+        "favorites_row_selected": [196, 239, 255, 255],
+
+        "monitor_text_inactive": [154, 154, 154, 255],
+        "monitor_background_inactive": [240, 240, 240, 255],
```

Comment:
If you're adding these, it's also up to you to ensure that the other built-in monitor options also use these variables.

Reply:
Yeah I've been replacing hardcoded colors and sizes when I see them... but someone didn't use any from the start haha. 😛